javaschool-backend/src/main/java/com/nearsoft/javaschoolbackend/util/RabbitMQSender.java

```diff
@@ -0,0 +1,33 @@
+package com.nearsoft.javaschoolbackend.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.nearsoft.javaschoolbackend.model.request.TypeRequest;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RabbitMQSender {
+
+    private final RabbitTemplate rabbitTemplate;
+
+    public RabbitMQSender(RabbitTemplate rabbitTemplate){
+        this.rabbitTemplate = rabbitTemplate;
+    }
+
+    @Value("${rabbitmq.exchange}")
```

Comment:
Checkout @ConfigurationProperties, preferable in Spring Boot
[https://www.baeldung.com/configuration-properties-in-spring-boot](url)